packages/labs/eleventy-plugin-lit/README.md

```diff
@@ -10,6 +10,16 @@ production. Breaking changes are likely to happen frequently. 🚧
 
 ## Usage
 
+```sh
```

Comment:
Let's not actually include this here -- since these are the steps for developing the plugin, rather than using it. Actually we've got a new README landing soon at https://github.com/lit/lit/pull/2488.

Reply:
reverted